modules/standard/SysBasic.chpl

```diff
@@ -215,151 +215,151 @@ inline proc EFORMAT return chpl_macro_int_EFORMAT():err_t;
 /* Argument list too long. The number of bytes used for the argument and
   environment list of the new process exceeded the current limit. (POSIX.1)
  */
-extern const E2BIG:err_t;
+extern const E2BIG:qio_err_t;
```

Comment:
These already exist in OS.POSIX and arguably should just be removed from here. But I have no problem with this PR just changing err_t -> qio_err_t.

Reply:
That's exciting since I think that takes care of 90% of the rest of SysBasic.  I'm definitely not going to take that on here, but filed https://github.com/chapel-lang/chapel/issues/19755 against it, and think it makes sense to do this as part of our review of SysBasic for 2.0.